Source/GameBaseFramework/Private/Input/GBFInputMappingContextStackItem.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Input/GBFInputMappingContextStackItem.h"
+
+#include <EnhancedInputSubsystems.h>
+
+#if WITH_EDITOR
+#include "DVEDataValidator.h"
+#endif
+
+void UGBFInputMappingContextStackItem::AddInputMappings( UEnhancedInputLocalPlayerSubsystem * input_system )
```

Comment:
I would name like in `UGBFAbilitySet`:

`AddToInputSystem`

`RemoveFromInputSystem`

Reply:
Done.